prelude.inc

```diff
@@ -1,7 +1,19 @@
 .set noat
 .set noreorder
 .set gp=64
+
 .macro glabel label
+    .global \label
+    .type \label, @function
```

Comment:
This will be a regression for existing projects using glabel for data, and it doesn't gain us anything: asm-processor already does some elf munging that adds ST_FUNC markers appropriately. It could be worth adding commented-out though together with a short note explaining this, in case people copy the prelude for use elsewhere.